eras/shelley/test-suite/cardano-ledger-shelley-test.cabal

```
@@ -165,7 +165,7 @@ test-suite cardano-ledger-shelley-test
     cardano-ledger-byron,
     cardano-ledger-core:{cardano-ledger-core, testlib},
     cardano-ledger-shelley:{cardano-ledger-shelley, testlib},
-    cardano-ledger-shelley-test >=1.1,
+    cardano-ledger-shelley-test >=1.6,
```

Why is there a bound on itself? It needs to be removed.
```suggestion
    cardano-ledger-shelley-test,
```